docs-client/src/containers/App/index.tsx

```diff
@@ -102,26 +107,31 @@ function AppDrawer({ classes, navigateTo, specification }: AppDrawerProps) {
             </ListItemText>
           </ListItem>
           {specification.getServices().map((service) => (
-            <li key={service.name}>
-              <ul>
-                <ListSubheader className={classes.methodHeader}>
+            <div key={service.name}>
+              <ListSubheader className={classes.methodHeader}>
+                <Typography variant="title">
```

Comment:
`subheading` looks better to me. What do you think?

Reply:
You're right, switched to it and also added `<code>`